<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeAutomatic</class>
 <widget class="QWidget" name="ModeAutomatic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,1,0,0,0,0,0,1,0">
     <item>
      <widget class="QLabel" name="label_38">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Автоматический режим</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_124">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget_missionPlanning">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_missionPlanning">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_50">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Выбор миссии</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_missionPlanning_goto">
           <property name="text">
            <string>Выход в точку</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_missionPlanning_cpp">
           <property name="text">
            <string>Покрытие акватории</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_missionPlanning_goto">
        <layout class="QVBoxLayout" name="verticalLayout_11" stretch="0,0,0,0,0,0">
         <item>
          <widget class="QLabel" name="label_53">
           <property name="text">
            <string>Введите параметры точки</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_missionPlanning_goto_x">
             <property name="text">
              <string>x, см</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_missionPlanning_goto_x">
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_55">
             <property name="text">
              <string>y, см</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_missionPlanning_goto_y">
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_56">
             <property name="text">
              <string>Радиус, см</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_missionPlanning_goto_r">
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_missionPlanning_goto_update">
           <property name="text">
            <string>Обновить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_missionPlanning_goto_clean">
           <property name="text">
            <string>Очистить цель</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_missionPlanning_goto_back">
           <property name="text">
            <string>Вернуться к выбору миссий</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_missionPlanning_cpp">
        <layout class="QVBoxLayout" name="verticalLayout_13" stretch="0,0,0,1,0,0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_63">
             <property name="text">
              <string>Межгалсовое расстояние</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_missionPlanning_cpp_distanceTack">
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_missionPlanning_cpp_on_off">
           <property name="text">
            <string>Расствить границы осмотра акватории</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_missionPlanning_cpp_zonaResearch"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QPushButton" name="pushButton_missionPlanning_cpp_make">
             <property name="text">
              <string>Построить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_missionPlanning_cpp_make_clean">
             <property name="text">
              <string>Очистить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_missionPlanning_cpp_back">
           <property name="text">
            <string>Вернуться к выбору миссий</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_missionPlanning_goto_on_trajectory">
       <property name="text">
        <string>Включить/выключить запись траектории</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_missionPlanning_goto_on_trajectory_clear">
       <property name="text">
        <string>Очистить траекторию</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_125">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_39">
         <property name="text">
          <string>Состояние агента</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QPushButton" name="pushButton_missionControl_modeIdle">
           <property name="text">
            <string>Ожидание</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_missionControl_modeStart">
           <property name="text">
            <string>Старт</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_missionControl_modeComplete">
           <property name="text">
            <string>Завершение</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_52">
         <property name="text">
          <string>Текущие состояние:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_missonStatus">
         <property name="text">
          <string>- Отсутствует</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_after">
       <property name="text">
        <string>Обратно</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
